<?xml version="1.0"?>
<!-- 
Ajay jain
July 1, 2014
ARSENL Lab, Naval Postgraduate School
-->
<launch>

	<!-- No namespace here as we will share this description. 
		 Access with slash at the beginning -->
    <param name="robot_description" command="$(find xacro)/xacro.py '$(find husky_description)/urdf/base.urdf.xacro'" />

	<!-- http://answers.ros.org/question/41433/multiple-robots-simulation-and-navigation/# -->
	<!-- BEGIN ROBOT 1-->
	<group ns="robot1">
		<param name="tf_prefix" value="robot1_tf" />
		<include file="$(find husky_pursuit)/launch/simul/one_husky.launch" >
			<arg name="init_pose" value="-x 1 -y 1 -z 0" />
			<arg name="robot_name"  value="Robot1" />
		</include>
	</group>

	<!-- BEGIN ROBOT 2-->
	<group ns="robot2">
		<param name="tf_prefix" value="robot2_tf" />
		<include file="$(find husky_pursuit)/launch/simul/one_husky.launch" >
			<arg name="init_pose" value="-x -3 -y 1 -z 0" />
			<arg name="robot_name"  value="Robot2" />
		</include>
	</group>
	
</launch>